### ISSUE TITLE
missing bower.json file with "main" key and dist directory with build package

### ISSUE THREAD

**Issue by rudzikdawid** [paradox41/angular-chartist.js#87]
could You add to repository bower.json file and dist directory with latetst build package?

bower.json
```
 ...
 "main": "./dist/angular-chartist.js"
 ...
```
Somethink like https://github.com/AngularClass/angular-websocket/blob/master/bower.json

**Comment by paradox41** [comment]
What version are you using @rudzikdawid ?

**Comment by rudzikdawid** [comment]
0.4.2

should i downgrade to 0.3.13  or other version?

**Comment by paradox41** [comment]
Those are very old versions. If you want to have bower support you should install [v3.3.13](https://github.com/paradox41/angular-chartist.js/releases/tag/v3.3.13). Bower support has been *dropped* as of version 4.x

**Comment by alfmatos** [comment]
Hey @paradox41, any particular reason for dropping bower support?

**Comment by paradox41** [comment]
@alfmatos Sure. The reason is that NPM has become the de-facto package manager for the frontend. Bower (along with Component, JSPM, etc) never really got enough mind share. Most of the packages I use in my day-to-day work use NPM exclusively, FWIW. 

Also, you can still get the 4.x line by installing [from git](http://stackoverflow.com/a/16951315). I just will no longer maintain a `bower.json`

**Comment by alfmatos** [comment]
@paradox41 much appreciated. Perhaps for legacy sake adding this to the documentation?

```bower install 'https://github.com/paradox41/angular-chartist.js.git#v4.2.0' --save```

**Comment by paradox41** [comment]
That sounds good. I can do that

**Comment by alfmatos** [comment]
@paradox41 I would, but at this point I'm not sure if that will solve anything, since the [bower entry for angular-chartist.js](https://libraries.io/bower/angular-chartist.js) is picking up the latest version regardless if `bower.json`exists.

This affects gulp builds using wiredep (which is what I'm stuck with), as wiredep only picks up  packages that have a `bower.json`.

I've added `bower.json`and `dist` using webpack on a [fork](https://github.com/alfmatos/angular-chartist.js), but it seems to hit a wall. For some reason the it fails to load the chartist lib.

```undefined is not an object (evaluating '_chartist2.default[this.chartType]')```

Any thoughts?

**Comment by paradox41** [comment]
Do you have Chartist loaded correctly?

**Comment by alfmatos** [comment]
I've added chartist as well throught: ```bower install chartist --save```

Wiredep picks up both:

```
<script src="../bower_components/chartist/dist/chartist.min.js"></script>
<script src="../bower_components/angular-chartist.js/dist/angular-chartist.js"></script>
```

I'm loading angular-chartist as a regular dependency:

```
angular
    .module('myapp', [
        ...,
        'angular-chartist']
```

**Comment by paradox41** [comment]
Can you add a `debugger;` in [here](https://github.com/alfmatos/angular-chartist.js/blob/master/dist/angular-chartist.js#L14) somewhere? just check to make sure all of [these guys](https://github.com/alfmatos/angular-chartist.js/blob/master/dist/angular-chartist.js#L13) are defined?

**Comment by alfmatos** [comment]
Right on. `_chartist` is undefined, as a result of ``global.chartist`` being undefined.

**Comment by paradox41** [comment]
So does chartist expose itself as globally available?

**Comment by alfmatos** [comment]
From the chartist code, `chartist 0.9.8` registers itself as Chartist. So the problem was with the variable naming. Using global.Chartist made it work.

Should this be fixed in your codebase, or something to take up with chartist upstream?

**Comment by paradox41** [comment]
Looks like a bug in my code. I will take a look today.

**Comment by paradox41** [comment]
Released as v4.2.1

**Comment by alfmatos** [comment]
@paradox41 Thanks a lot for the wonderful work and support.
